requirements.txt

```diff
@@ -10,8 +9,6 @@ jsonschema>=2.5.1
 makefun>=1.7.0,<2
 marshmallow>=3.7.1,<4.0.0
 mistune>=0.8.4
-nbformat>=5.0
```

Comment:
Used here and there in tests - we should remove completely but for now moving to dev deps